client/src/components/photoupload.js

```diff
@@ -3,6 +3,7 @@ import axios from "axios";
 import "../App.css";
 import { Button, Modal, Form } from "react-bootstrap";
 
+
 class RenderProfilePage extends Component {
   constructor(props) {
     super(props);
```

Comment:
Just try to avoid putting random spaces like these, or at least remove them before making a request, because this will create merge conflicts for me when I try to pull master after this

Reply:
yes bröther